react/components/ProductImages/components/Carousel/ThumbnailSwiper.js

```diff
@@ -30,25 +31,27 @@ const Thumbnail = ({
       onClick={() => gallerySwiper && gallerySwiper.slideTo(index)}
     >
       <figure
-        className={styles.figure}
+        className={handles.figure}
         itemProp="associatedMedia"
         itemScope
         itemType="http://schema.org/ImageObject"
       >
         <img
-          className="w-100 h-auto db"
+          className={`${handles.thumbImg} w-100 h-auto db`}
           itemProp="thumbnail"
           alt={alt}
           src={imageUrl(thumbUrl, THUMB_SIZE, THUMB_MAX_SIZE, aspectRatio)}
         />
       </figure>
       <div
-        className={`absolute absolute--fill b--solid b--muted-2 bw1 ${styles.carouselThumbBorder}`}
+        className={`absolute absolute--fill b--solid b--muted-2 bw0 ${handles.carouselThumbBorder}`}
       />
     </div>
   )
 }
 
+const CSS_HANDLES = ['productImagesThumb', 'productImagesThumbActive', 'figure', 'thumbImg', 'carouselThumbBorder', 'carouselGaleryThumbs']
```

Comment:
Nitpick: moving this to after the `import` statements would make it a bit more evident